types/types.ts

```diff
@@ -1,6 +1,8 @@
 import { Zora20210101 } from './zora/20210101';
 import { Catalog20210202 } from './catalog/20210202';
+import { Amulet20210221 } from './amulet/20210221';
```

Comment:
oy, I need to refactor this...

but you will need to add Amulet20210228 here for the type to be exported!

Reply:
I will note that for the future! I am sticking with just 20210221 so, this is moot for now.